api/app/Console/Commands/CheckIntl.php

```diff
@@ -158,7 +158,7 @@ private function checkStrings($enum)
 
         // Find exact matches of strings across locales
         $intersectLocales = array_filter(array_keys(array_intersect_assoc($existing['en'], $existing['fr'])), function ($item) {
-            return ! is_array($item) && ! in_array($item, $this->allowedMatch);
+            return ! in_array($item, $this->allowedMatch);
         });
```

Comment:
The variable apparently would never be an array making the deleted `! is_array($item)` always true